src/rgw/rgw_rest_pubsub.cc

```diff
@@ -195,8 +195,8 @@ class RGWPSCreateTopicOp : public RGWOp {
       return 0;
     }
     if (ret == 0) {
-      if (result.user == s->owner.id ||
-          !s->cct->_conf->mandatory_topic_permissions) {
+      if (result.user.to_str().length() == 0 ||
```

Comment:
can you please use `verify_topic_owner_or_policy()` here?

Reply:
OK